database/src/service/read.rs

```diff
@@ -106,23 +121,15 @@ impl DatabaseReadHandle {
         }
     }
 
-    /// TODO
+    /// Access to the actual database environment.
+    ///
+    /// TODO: we need this for testing but should we allow it publicly?
+    /// Or within `crate`? It allows anyone to start tampering with the
+    /// database directly instead of going through `service`.
     #[inline]
     pub const fn env(&self) -> &Arc<ConcreteEnv> {
         &self.env
     }
```

Comment:
Should the `DatabaseReadHandle` (the read service) give access to the actual database?

It lets anybody outside of `cuprate_database` directly use the database instead of going through `service`.

Reply:
we could but I would rather not, probably should put a warning on it though